pages/osx/as.md

```diff
@@ -6,16 +6,16 @@
 
 - Assemble a file, writing the output to `a.out`:
 
-`as {{file.s}}`
+`as {{path/to/file.s}}`
 
 - Assemble the output to a given file:
 
-`as {{file.s}} -o {{out.o}}`
+`as {{path/to/file.s}} -o {{path/to/out.o}}`
```

Comment:
```suggestion
`as {{path/to/file.s}} -o {{path/to/output_file.o}}`
```
How about this?

or
```suggestion
`as {{path/to/file.s}} -o {{path/to/output.o}}`
```

Reply:
The second suggestion is not valid from the converter perspective: `file` keyword should be passed too.